components/table/style/index.less

```diff
@@ -233,6 +233,20 @@
         background: @table-row-hover-bg;
       }
     }
+    &.@{table-prefix-cls}-row-selected > td {
+      background: @table-selected-row-bg;
```

Comment:
This is already defined and no need duplicate define: https://github.com/ant-design/ant-design/blob/master/components/table/style/index.less#L297

Reply:
removed duplicate code